lib/parse-number.c

```diff
@@ -146,6 +147,12 @@ _process_suffix(const gchar *suffix, gint64 *d)
   return TRUE;
 }
 
+static gboolean
+_no_suffix(const gchar *suffix)
+{
+  return (*suffix == '\0' || isspace(*suffix));
```

Comment:
Did you happen to consider `g_ascii_isspace()` for this purpose?

Reply:
No, but I can use it.